VERSION

```diff
@@ -1 +1 @@
-0.063
```

Comment:
We've been gradually migrating all the version numbers to use semver syntax so this is a good opportunity to bump this to ... `0.1.0` which doesn't really mean much beyond "bigger than "0.063"

Reply:
`0.064` begone; it's all semver from now on. Also fixed in commit 992be8288313f0f8b8b74fc1fab1444dc0f35fdf